modules/sdp/SDP.js

```diff
@@ -86,9 +89,9 @@ export default class SDP {
      * @returns {boolean}
      */
     containsSSRC(ssrc) {
-        const souceMap = this.getMediaSsrcMap();
+        const sourceMap = this.getMediaSsrcMap();
 
-        return Object.values(souceMap).some(media => media.ssrcs[ssrc]);
+        return [ ...sourceMap.values() ].some(media => media.ssrcs[ssrc]);
```

Comment:
Fun observation: Iterators now have `some` but only on very recent browsers :-/

Reply:
Good to know, I will look that up.

Reply:
Actually, it may work today, since Babel uses core-js to polyfill all missing ES features. Can you please give it a try? It should basically be: ` return sourceMap.values().some(media => media.ssrcs[ssrc]);`

Reply:
I wouldn't worry too much about it here in this specific instance since this method is going away in my follow up PR but I will keep this in mind.